### ISSUE TITLE
Support for Tailwind's dark mode

### ISSUE THREAD

**Issue by ad-si** [matheus23/elm-default-tailwind-modules#9]
It would be cool if there was a way to use Tailwind's dark mode:
https://tailwindcss.com/docs/dark-mode

**Comment by kutyel** [comment]
Right, from https://html-to-elm.com/ I'm getting `Bp.dark` when tailwind uses that selector, but obviously that is not a breakpoint, so I guess many libs would need to change to fix this 🙏🏻
